example/quarkus-jvm.Dockerfile

```diff
@@ -8,7 +8,13 @@ ENV RUNTIME_TYPE quarkus
 # 4. Build the image: docker build --tag quay.io/<yournamespace>/process-quarkus-example:latest -f quarkus-jvm.Dockerfile .
 #   4.1 Optionally test the image locally: docker run --rm -it -p 8080:8080 quay.io/<yournamespace>/process-quarkus-example:latest
 # 5. Push it: docker push quay.io/<yournamespace>/process-quarkus-example:latest
-# 6. Deploy it on Kubernetes with the kogito operator, as a reference use process-quarkus-example-runtime.yaml
+# 6. Deploy it on Openshift with the kogito operator, as a reference use process-quarkus-example-runtime.yaml
```

Comment:
Do we support just OpenShift for RHPAM operator or also other Kubernetes implementations?

Reply:
OKE as well, looks like.

Reply:
Does OKE means `Oracle Cloud Infrastructure Container Engine`?

Reply:
https://docs.openshift.com/container-platform/4.3/welcome/oke_about.html

Reply:
@sutaakar I think the idea is to support other k8s implementations.